projects/rocprofiler-systems/docker/Dockerfile.ubuntu.ci

```
@@ -27,7 +27,7 @@ RUN apt-get update && \
         bzip2 chrpath cmake curl environment-modules flex gettext git-core gnupg2 \
         gzip iproute2 libiberty-dev libpapi-dev libpfm4-dev libsqlite3-dev libtool \
         locales lsb-release m4 ninja-build python3-pip software-properties-common \
-        texinfo unzip wget vim zip zlib1g-dev && \
+        texinfo unzip wget vim zip zlib1g-dev nlohmann-json3-dev && \
```

```suggestion
        locales lsb-release m4 ninja-build nlohmann-json3-dev python3-pip \
        software-properties-common texinfo unzip vim zip zlib1g-dev && \
```
Sorting list alphabetically.